tools/bio3d/macros.xml

```diff
@@ -0,0 +1,14 @@
+<macros>
+    <token name="@VERSION@">2.3</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="2.3_3">r-bio3d</requirement>
+            <yield/>
```

Comment:
@tsenapathi this yield is very important. Otherwise, your r-lattice package is never installed. With this in place, I get no R error anymore. But the outputs file still don't match 100%. Which can be due to floating point differences. Can you check this please?